rotkehlchen/db/evmtx.py

```diff
@@ -271,14 +272,18 @@ def get_transaction_hashes_not_decoded(
             self,
             chain_id: Optional[ChainID],
             limit: Optional[int],
+            addresses: Optional[List[ChecksumEvmAddress]],
     ) -> List[EVMTxHash]:
         """Get transaction hashes for the transactions that have not been decoded.
         Optionally by chain id.
         If the limit argument is provided then it is used in the SQL query with
         the default order.
         """
         querystr = 'SELECT A.tx_hash from ' + TRANSACTIONS_MISSING_DECODING_QUERY
```

Comment:
Question. Should this function return transactions that do have receipts queried but are not decoded yet or both (wither when there is no receipt or when the receipt is queried but the transaction is not decoded)?

Reply:
For that we have `get_transaction_hashes_no_receipt`. The usage of this functions is to decode the transactions that have not been decoded yet but we have the receipts already